howto/website-guidelines-checklist.md

```diff
@@ -0,0 +1,41 @@
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md)
+(**Note**, not all of these are applicable to all projects):
+
+- [ ] 1. Website should be [hosted in an open source repo](./repo-setup.md)
+  - [ ] Hosted in the same organization as the main project
+- [ ] 2. Reference the origin company correctly (if needed)
+  *Note*: It is OK to say that, e.g., “Prometheus was originally created by 
+  Soundcloud” or “Kubernetes builds upon 15 years of experience of running 
+  production workloads at Google,” but the origin company should not otherwise
+  be referred to on the project homepage.
+- [ ] 3. No links or forms for capturing enterprise support leads should be present
+  It is fine to have an enterprise support, commercial partners or similar page.
+  - [ ] If page is present, the companies list is alphabetized or randomized on load.
+  - [ ] If page is present, the vetting of companies listed is complete
+    *Note*: Projects are welcome to outsource this vetting to CNCF staff if it
+    becomes a burden.
+- [ ] 4. Links to companies offering support go to a page that at least mentions
+  support of the project
+- [ ] 5. Copyright notice present at bottom of page.
+  Copyright to the project authors or to CNCF, not the origin company.
+- [ ] 6. CNCF Branding elements
+  - [ ] “We are a Cloud Native Computing Foundation project.” or “We are a Cloud
+  Native Computing Foundation sandbox project.” present (depending on status)  
+  - [ ] CNCF logo near the bottom of their project homepage  
+  - [ ] (optional) link to KubeCon + CloudNativeCon as the events approach  
+- [ ] 7. Footer contains:
```

Comment:
I need to review this for accuracy, there's something that tells me this is maybe old information. 
I cannot put my finger on it.

Reply:
Thanks for  looking at it -- I do have to do another pass doing a close read of the foundation's guidelines

Reply:
@amye - I'm going to merge this (for now) and we can update later.